src/05-excitation-transfer.md

```diff
@@ -0,0 +1,762 @@
+
+Thinking more carefully about these unfamiliar horizontal levels, there are 2 ways we can combine $|n,+,-\rangle$ and $|n,-,+\rangle$ together:
+- $|n,+,-\rangle + |n,-,+\rangle$ 
+- $|n,+,-\rangle  - |n,-,+\rangle$ 
+
+These combinations are reminiscent of the "in phase" and "out of phase" states that we described [back in tutorial 1](https://nbviewer.jupyter.org/github/project-ida/two-state-quantum-systems/blob/master/01-an-isolated-two-state-system.ipynb#1.2-Coupling-between-two-states-of-the-same-energy).  Because there are two combinations, we can expect that each horizontal line in Fig 6 is in fact 2 horizontal lines on top of each other, i.e:
+
+ - $|1,+,-\rangle \pm |1,-,+\rangle $ - green/red
+ - $|3,+,-\rangle \pm |3,-,+\rangle $ - pink/grey
+
+What else can we say about Fig 6?
+
+When looking at where the levels cross, there appear to be two different situations:
+- 2 levels come together - we've seen this before and we expect these to form anti-crossings when we switch the coupling on (i.e. $U\neq 0$)
+- 4 levels come together - we've not encountered this before so we need to explore it further
+
+From now on we'll continue our exploration using only even parity, i.e Fig 6. 
+
+Let's switch on the coupling and see if there are any surprises.
+
+
+## 5.6 - Crossings and anti-crossings
+
+
+We'll create a relatively strong coupling to start with i.e. $U=0.1$. This way, we can easily spot any changes in the energy level diagram compard to Fig 6.
+
+```python
+# We re-use operators that made Fig 6 (these had even parity)
+for i, row in df_even.iterrows():
+    H =  row["$\Delta E$"]*two_state_1+ row["$\Delta E$"]*two_state_2 + 1*bosons + 0.1*interaction_1 + 0.1*interaction_2
+    evals, ekets = H.eigenstates()
+    df_even.iloc[i,1:] = evals 
+```
+
+```python
+df_even.plot(x="$\Delta E$",ylim=[-0.5,5.5],legend=True, 
+        title=f"Even energy levels for {H_latex}   ($\omega=1$, $U=0.1$)   (Fig 7)",
+             figsize=(10,8));
+
+plt.ylabel("Energy");
+
+```
+
+There are many things we can see in Fig 7:
+1. Where 2 levels have come together (e.g. levels 4 and 5) we see an anti-crossing as we have come to expect from interacting levels
+2. The horizontal levels of Fig 6 have split into 2 levels as we suspected. This confirms that we do indeed have 4 levels coming together at some anti-crossings, e.g. levels 1,2,3,4
+3. There appear to still be genuine crossings between some levels, most strikingly seen at the intersection of levels 2 and 3 - this indicates there might be non interacting sub-universes with each parity universe 🤔
```

Comment:
here actually red and green seems to anticross because of the wrong plotting rendering (so if we point is out here we should mention this)

Reply:
Yes, I understand what you mean, but compared to the other two levels (1 and 4) around DeltaE=1 they look very much like they cross. It's visually the same kind of thing we say is a crossing in Figs 5 and 6. I've added some extra text to bring peoples' attention to DeltaE=1 region.